main/src/ca/uwaterloo/flix/language/phase/jvm/GenTupleClasses.scala

```diff
@@ -139,13 +121,7 @@ object GenTupleClasses {
       val fieldName = s"field$ind"
 
       // Defining fields of the tuple
-      AsmOps.compileField(visitor, fieldName, field, isStatic = false, isPrivate = true)
-
-      // Emitting getter for each field
-      AsmOps.compileGetFieldMethod(visitor, classType.name, fieldName, s"getIndex$ind", field)
```

Comment:
Can AsmOps.compileGetFieldMethod be removed or is it used in other places?

Reply:
Its used in many of the other gen classes